src/main/java/mat/client/export/measure/ManageMeasureExportView.java

```diff
@@ -223,4 +238,14 @@ public boolean isCompositeMeasurePackageRadio() {
 	public Button getMeasureNameLink() {
 		return measureNameLink;
 	}
+
+	@Override
+	public boolean isXml() {
+		return xmlRadio.getValue();
+	}
+
+	@Override
+	public boolean isAll() {
+		return allRadio.getValue();
```

Comment:
Looks like you use 'TABS' instead of 'SPACES' for formatting your code

Reply:
![image](https://user-images.githubusercontent.com/57495404/76354446-d7c26f00-62e8-11ea-8311-bba3d97cd44f.png)
It looks good to me in local.